packages/redux/src/analytics/middlewares/__tests__/setUser.test.ts

```diff
@@ -289,7 +318,7 @@ describe('setUserMiddleware', () => {
 
       await dispatchUserChangingAction(
         store,
-        Array.from(triggerSetUserActions)[0],
+        Array.from(triggerSetUserActions)[0] || '',
```

Comment:
This should not be done as it changes the testing code. If you need to silence the typescript error, you can do this instead:

```suggestion
        Array.from(triggerSetUserActions)[0] as string,
```